tests/test_transforms/test_outliers_transform.py

```diff
@@ -48,3 +48,26 @@ def test_outliers_detection(transform, method, outliers_tsds):
         nan_timestamps = detectiom_method_results[segment]
         transformed_column = transformed_df.loc[non_nan_index[segment], pd.IndexSlice[segment, "target"]]
         assert np.all(transformed_column[transformed_column.isna()].index == nan_timestamps)
+
+
+@pytest.mark.parametrize(
+    "transform",
+    [
+        MedianOutliersTransform(in_column="target"),
+        DensityOutliersTransform(in_column="target"),
+        SAXOutliersTransform(in_column="target"),
+    ],
+)
+def test_inverse_transform(transform, example_tsds):
```

Comment:
it would be better to split this test into two ones: one for train timestamps and one for future timestamps; this way each test covers one use case and can show the problem clearly